Formula/metacall.rb

```diff
@@ -0,0 +1,105 @@
+class Metacall < Formula
+  desc "Ultimate polyglot programming experience"
+  homepage "https://metacall.io"
+  url "https://github.com/metacall/core/archive/refs/tags/v0.5.27.tar.gz"
+  sha256 "1493afadac893b7b3674ea44dd7caa7fd74989880658ae23003b30d3721bb92d"
+  license "Apache-2.0"
+  head "https://github.com/metacall/core.git", branch: "develop"
+
+  depends_on "cmake" => :build
+  depends_on "node@14"
+  depends_on "openjdk"
+  depends_on "python@3.9"
+  uses_from_macos "ruby"
+
+  def install
+    Dir.mkdir("build")
+    Dir.chdir("build")
+    args = std_cmake_args + %W[
+      -Wno-dev
+      -DCMAKE_BUILD_TYPE=Release
+      -DOPTION_BUILD_SCRIPTS=OFF
+      -DOPTION_FORK_SAFE=OFF
+      -DOPTION_BUILD_TESTS=OFF
+      -DOPTION_BUILD_EXAMPLES=OFF
+      -DOPTION_BUILD_LOADERS_PY=ON
+      -DOPTION_BUILD_LOADERS_NODE=ON
+      -DNodeJS_INSTALL_PREFIX=#{buildpath}
+      -DOPTION_BUILD_LOADERS_JAVA=ON
+      -DOPTION_BUILD_LOADERS_JS=OFF
+      -DOPTION_BUILD_LOADERS_C=OFF
+      -DOPTION_BUILD_LOADERS_COB=OFF
+      -DOPTION_BUILD_LOADERS_CS=OFF
+      -DOPTION_BUILD_LOADERS_RB=ON
+      -DOPTION_BUILD_LOADERS_TS=ON
+      -DOPTION_BUILD_LOADERS_FILE=ON
+      -DOPTION_BUILD_PORTS=ON
+      -DOPTION_BUILD_PORTS_PY=ON
+      -DOPTION_BUILD_PORTS_NODE=ON
+    ]
+    system "cmake", *args, ".."
+    system "cmake", "--build", ".", "--target", "install"
+
+    shebang = "\#!/usr/bin/env bash\n"
+    # debug = "set -euxo pipefail\n"
+
+    metacall_extra = [
+      "LOC=#{prefix}\n",
+      "export LOADER_LIBRARY_PATH=\"#{lib}\"\n",
+      "export SERIAL_LIBRARY_PATH=\"#{lib}\"\n",
+      "export DETOUR_LIBRARY_PATH=\"#{lib}\"\n",
+      "export PORT_LIBRARY_PATH=\"#{lib}\"\n",
+      "export CONFIGURATION_PATH=\"#{prefix}/configurations/global.json\"\n",
```

Comment:
This should be in `etc`

Reply:
is it because they are config files, upstream they are set this way at the moment, I don't think I can get upstream to change this.

Reply:
@SMillerDev as I said on the previous post, how can be this on etc? The launcher needs extra functionality a part of the environment variables. I can link you some other examples of the launcher for other platforms:

Windows relocatable distributable: https://github.com/metacall/distributable-windows/blob/master/metacall.bat
Linux Guix based portable distributable (as you can see, this one is auto-generated in the install phase): https://github.com/metacall/install/blob/a30f35ccb87387785971f57d06c4a1982a3de677/install.sh#L372

Reply:
It's a config file so it should be in etc. Windows or guix don't have much to do with filesystem standards.